src/main/java/com/sprint/findex_team6/repository/IndexValRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.sprint.findex_team6.repository;
+
+import com.sprint.findex_team6.entity.IndexVal;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface IndexValRepository extends JpaRepository<IndexVal, Long> {
+  List<IndexVal> findByIndexId(Long indexId);
```

Comment:
```suggestion
  List<IndexVal> findByIndex_Id(Long indexId);
```
인덱스 id로 IndexVal를 찾는 메서드가 맞을까요? 맞다면 이렇게 작성해야 합니다.

Reply:
네 확인했습니다~!